sources/qa-lang-main.php

```diff
@@ -145,10 +145,10 @@
        'selected'=>'izbrano', 
        'send_button'=>'Pošlji', 
        'since_x'=>'od ^', //since ^
-       'suggest_ask'=>'Pomagaj nam začeti in ^1asking question^2.', //Help get things started by ^1asking a question^2.
-       'suggest_category_qs'=>'Če si želite ogledati več, kliknite za vse ^1questions v tej category^2.', //To see more, click for all the ^1questions in this category^2.
-       'suggest_qs'=>'Če si želite ogledati več kliknite za ^1full seznam questions^2.', //To see more, click for the ^1full list of questions^2.
-       'suggest_qs_tags'=>'Če si želite ogledati več kliknite za ^1full seznam questions^2 ali ^3popular tags^ 4.', //To see more, click for the ^1full list of questions^2 or ^3popular tags^4.
+       'suggest_ask'=>'Pomagaj nam začeti z ^1asking a question^2.', //Help get things started by ^1asking a question^2.
+       'suggest_category_qs'=>'Če si želite ogledati več, kliknite za vse ^1questions in this category^2.', //To see more, click for all the ^1questions in this category^2.
+       'suggest_qs'=>'Če si želite ogledati več kliknite za ^1full list of questions^2.', //To see more, click for the ^1full list of questions^2.
+       'suggest_qs_tags'=>'Če si želite ogledati več kliknite za ^1full list of questions^2 ali ^3popular tags^ 4.', //To see more, click for the ^1full list of questions^2 or ^3popular tags^4.
```

Comment:
Vse spremembe so zbrane v patch 4, ostali niso potrebni.


2015-04-13 22:02 GMT+02:00 Matjaž M. <notifications@github.com>:

> In sources/qa-lang-main.php
> <https://github.com/matmr/q2a-prevajanje-FS/pull/9#discussion_r28273315>:
>
> > @@ -145,10 +145,10 @@
> >         'selected'=>'izbrano',
> >         'send_button'=>'Pošlji',
> >         'since_x'=>'od ^', //since ^
> > -       'suggest_ask'=>'Pomagaj nam začeti in ^1asking question^2.', //Help get things started by ^1asking a question^2.
> > -       'suggest_category_qs'=>'Če si želite ogledati več, kliknite za vse ^1questions v tej category^2.', //To see more, click for all the ^1questions in this category^2.
> > -       'suggest_qs'=>'Če si želite ogledati več kliknite za ^1full seznam questions^2.', //To see more, click for the ^1full list of questions^2.
> > -       'suggest_qs_tags'=>'Če si želite ogledati več kliknite za ^1full seznam questions^2 ali ^3popular tags^ 4.', //To see more, click for the ^1full list of questions^2 or ^3popular tags^4.
> > +       'suggest_ask'=>'Pomagaj nam začeti z ^1asking a question^2.', //Help get things started by ^1asking a question^2.
> > +       'suggest_category_qs'=>'Če si želite ogledati več, kliknite za vse ^1questions in this category^2.', //To see more, click for all the ^1questions in this category^2.
> > +       'suggest_qs'=>'Če si želite ogledati več kliknite za ^1full list of questions^2.', //To see more, click for the ^1full list of questions^2.
> > +       'suggest_qs_tags'=>'Če si želite ogledati več kliknite za ^1full list of questions^2 ali ^3popular tags^ 4.', //To see more, click for the ^1full list of questions^2 or ^3popular tags^4.
>
> Pomaga. Kar je pomembno -- ali so v patch 4 vse spremebe, ali je treba
> dopolnit še z drugimi patchi, in če ja, katerimi.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matmr/q2a-prevajanje-FS/pull/9/files#r28273315>.
>